flow-model-generator/modules/flow-model-generator-core/src/main/java/io/ballerina/flowmodelgenerator/core/db/model/FunctionResult.java

```diff
@@ -43,5 +44,6 @@ public record FunctionResult(
         String version,
         String resourcePath,
         Function.Kind kind,
-        Integer returnError) {
+        Integer returnError,
+        Integer inferredReturnType) {
```

Comment:
Why this is an integer instead of a boolean?

Reply:
There is no boolean data type sqllite.

Reply:
We initially made this decision because SQLite doesn't support boolean values, requiring us to use integers to represent flags. However, looking into this discovered that the SQL driver manager automatically interprets integer values as flags.  Addressed with: 81b8bb3d9c727e99c1c497415d675f01c4b7ac72